reports/Kentucky.md

```diff
@@ -119,3 +119,15 @@ id: ky-louisville-7
 **Links**
 
 * [Video taken from across the street](https://www.reddit.com/r/Louisville/comments/gw5joa/peaceful_protesters_marching_in_st_mathews_get/)
+
+### Police dogpiling a protestor and then targets a witness | June 16th
```

Comment:
I think your subject-verb agreement is off here. Maybe "Police tackle protestor, then target witness"

Reply:
Thanks, I've corrected it.